go-controller/pkg/ovn/master.go

```diff
@@ -287,6 +287,33 @@ func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 		}
 	}
 
+	// FIXME: When https://github.com/ovn-org/libovsdb/issues/235 is fixed,
+	// use IsTableSupported(nbdb.LoadBalancerGroup).
+	if _, _, err := util.RunOVNNbctl("--columns=_uuid", "list", "Load_Balancer_Group"); err != nil {
+		klog.Warningf("Load Balancer Group support enabled, however version of OVN in use does not support Load Balancer Groups. " +
+			"Disabling Load Balancer Groups")
+		oc.loadBalancerGroupSupport = false
+	} else {
+		loadBalancerGroup := nbdb.LoadBalancerGroup{
+			Name: types.ClusterLBGroupName,
+		}
+		opModels := []libovsdbops.OperationModel{
+			{
+				Model:          &loadBalancerGroup,
+				ModelPredicate: func(lbg *nbdb.LoadBalancerGroup) bool { return lbg.Name == types.ClusterLBGroupName },
+				OnModelUpdates: []interface{}{
+					&loadBalancerGroup.Name,
+				},
+				ErrNotFound: false,
+			},
+		}
+		_, err = oc.modelClient.CreateOrUpdate(opModels...)
+		if err != nil {
+			klog.Errorf("Error creating cluster-wide load balancer group (%v)", err)
+			oc.loadBalancerGroupSupport = false
```

Comment:
you should not set this to false, and just return error here.

Reply:
Right, I updated it now.